doc/Type/List.pod6

```diff
@@ -17,6 +17,42 @@ Arrays to have every value of the list stored in a container.
 C<List> implements C<Positional> and as such provides support for
 L<subscripts|/language/subscripts>.
 
+=head1 Immutability
+
+Lists are immutable objects, i.e., neither the number of elements in a list
+nor the elements themselves can be changed. Thus, it is not possible to use
+operations that change the list structure itself such as L<shift|/routine/shift>,
+L<unshift|/routine/unshift>, L<push|/routine/push>, L<pop|/routine/pop>,
+L<splice|/routine/splice>
+and L<binding|/language/operators#index-entry-Binding_operator>.
+
+=begin code :lang<text>
+(1, 2, 3).shift;      # Error Cannot call 'shift' on an immutable 'List'
+(1, 2, 3).unshift(0); # Error Cannot call 'unshift' on an immutable 'List'
+(1, 2, 3).push(4);    # Error Cannot call 'push' on an immutable 'List'
+(1, 2, 3).pop;        # Error Cannot call 'pop' on an immutable 'List'
+(1, 2, 3)[0]:delete;  # Error Cannot remove elements from a List
+(1, 2, 3)[0] := 0;    # Error Cannot use bind operator with this left-hand side
+(1, 2, 3)[0] = 0;     # Error Cannot modify an immutable Int
+=end code
+
+A C«List» doesn't L«containerize|/language/containers» its elements, but if
+any element happens to be inside a L«Scalar|/type/Scalar» container then
+the element's contents can be replaced via an assignment.
+
+=begin code :lang<text>
```

Comment:
Same here. Please use the most adequate environment. If it's going to error, just skip-test